src/funcade/middleware/buddy.clj

```diff
@@ -61,5 +64,7 @@
 (defn jwks-authenticator
   "JSON Web Key Set specific:
    i.e. needs a 'https://foo.com/bar/jwks' URI that returns unsign keys"
-  [{:keys [uri] :as options}]
-  (jwks-backend (assoc options :keyset (jk/jwks->keys uri))))
+  [{:keys [uri refresh-keyset?] :as options}]
+  (jwks-backend (cond-> options
+                       (not refresh-keyset?)  (assoc :keyset (jwks/jwks->keys uri))
+                       refresh-keyset?        (assoc :keyset-fn (jwks/jwks->keys-fn uri)))))
```

Comment:
refreshing keyset should be an internal ability of funcade
it needs to take an interval (i.e. number of seconds)
and on every interval it just `swap!`s the atom with a new values